src/JsonSchema/Uri/UriRetriever.php

```diff
@@ -32,6 +32,14 @@ class UriRetriever implements BaseUriRetrieverInterface
         '|^https?://json-schema.org/draft-(0[34])/schema#?|' => 'package://dist/schema/json-schema-draft-$1.json'
     );
 
+    /**
+     * @var array A blacklist for media type ceheck exclusion
+     */
+    protected $mediaTypeBlacklist = array(
```

Comment:
More descriptive name please - perhaps something like 'allowInvalidContentType'.

Reply:
@erayd As I said I'm not happy with the name either and open for suggestions. `allowInvalidContentType` doesn't match the content either I believe.

How about `allowedInvalidContentTypeEndpoints`?

Reply:
That works :+1: